src/runtime/super.cpp

```diff
@@ -128,10 +129,10 @@ Box* superGetattribute(Box* _s, Box* _attr) {
         }
     }
 
-    Box* r = typeLookup(s->cls, std::string(attr->s), NULL);
-    // TODO implement this
-    RELEASE_ASSERT(r, "should call the equivalent of objectGetattr here");
-    return processDescriptor(r, s, s->cls);
+    Box* r = typeLookup(s->cls, attr_s, NULL);
+    if (r)
+        return processDescriptor(r, s, s->cls);
+    raiseExcHelper(AttributeError, "'super' object has has no attribute '%s'", attr->data());
```

Comment:
Oh, sorry yeah that's what CPython does.  I think the code here is the same as getattrInternalGeneric when there aren't any instance attributes, but I'm not sure if either of us are very confident in that.